CHANGELOG.md

```diff
@@ -1,5 +1,85 @@
 # CHANGELOG
 
+## 3.0.0
+
+This release adds pagination to ArsArsenal. In the process of doing this, we've made some breaking changes to the way URLs are constructed. For most users, this upgrade process should be minimal:
+
+### `makeURL` is now `listUrl` and `showUrl`
+
+The old `makeURL` option relied on a null check to determine if the requested url is for a list of items or a single record. To avoid that ambiguity, there are now two endpoints for URL construction:
+
+Instead of:
+
+```javascript
+function makeURL(url, slug) {
+  if (slug == null) {
+    return url
+  } else {
+    return `${url}/${slug}`
+  }
+}
+
+ArsArsenal.render({ makeURL })
+```
+
+Change this to:
+
+```javascript
+let listUrl = url => url
+let showUrl = (url, slug) => `${url}/${slug}`
+
+ArsArsenal.render({ listUrl, showUrl })
+```
+
+**These are the default implementations of each option.** We anticipate
+that this change affects very few users.
+
+### `makeQuery` is now `buildQuery` and returns an object
```

Comment:
I believe this is now `listQuery`

Reply:
and if that's the case, there are a couple of additional references to do a quick search/replace on